includes/polling/ports.inc.php

```diff
@@ -578,10 +588,12 @@
         if ($port_association_mode != "ifIndex") {
             $port['update']['ifIndex'] = $ifIndex;
         }
-
-        $port['update']['poll_time'] = $polled;
-        $port['update']['poll_prev'] = $port['poll_time'];
-        $port['update']['poll_period'] = $polled_period;
+        
+        if ($last_poll_time != $port['poll_time']) {
```

Comment:
Can this code block be deleted?

Reply:
I think we need to keep that
// set $port['update'] times if different from $device_global_ports
// if $port['update'] times not set and $port['update'] is void, value would be replaced by

Reply:
Can't you just update your $port_ids array?